frontend/src/index.tsx

```diff
@@ -74,13 +74,17 @@ const apolloClient = new ApolloClient({
   cache: new InMemoryCache(),
 });
 
-ReactDOM.render(
+const root = ReactDOM.createRoot(
+  document.getElementById("root") as HTMLElement,
+);
+root.render(
   <React.StrictMode>
-    <ApolloProvider client={apolloClient}>
-      <App />
-    </ApolloProvider>
+    <ChakraProvider>
```

Comment:
From what I've seen, the Chakra provider is usually set up in `App.tsx` instead. Is there a specific reason you chose to add it here instead?